### PR TITLE
Workable tab

### PR DESCRIPTION
**Pull request by f-hafner**

### CODE REVIEW

**Comment by f-hafner** [comment]
@maltelueken , I fixed the bug we discussed, added docs, and unit tests for `single_disease_server`. for `disease_selection_server`, I'd need to look how to mock the return of the child servers, and possibly refactor some of the code in the module into a single function (for easier mocking). let me know if you want me to do this.

File: inst/application/app.R [hunk 1]
```diff
@@ -1,4 +1,20 @@
 
+#disease_list <- list(
```
Comment (maltelueken) [inline inst/application/app.R:2]: Is this commented out code still needed?

**Review by maltelueken — APPROVED** [review]
Looks good! Maybe squash the commits again?

When wrapping up the package, we should rethink the exports. I don't think the user ever needs to use the different shiny modules, so we don't need to export them.

File: inst/application/app.R (hunk 1 above)
Comment (f-hafner) [reply inst/application/app.R:2]: nope, removed it

**Pull request merged**